.buildkite/hooks/post-command

```diff
@@ -2,12 +2,34 @@
 
 set +u
 
-if [[ $BUILDKITE_LABEL =~ ":selenium:" ]] || [[ $BUILDKITE_LABEL =~ ":docker: Build Image" ]];
-then
+if [[ $BUILDKITE_LABEL =~ ":selenium:" ]] || [[ $BUILDKITE_LABEL =~ ":docker: Build Image" ]]; then
   CONTAINERS=$(docker ps -a -q)
-  if [[ ${CONTAINERS} != "" ]];
-  then
+  if [[ ${CONTAINERS} != "" ]]; then
     echo "--- :docker: Clean environment"
     docker rm -f ${CONTAINERS}
   fi
+fi
+
+if [[ $BUILDKITE_BRANCH == "master" ]] || [[ $BUILDKITE_BRANCH =~ ^v.* ]] && [[ $BUILDKITE_PULL_REQUEST == "false" ]]; then
+  if [[ $BUILDKITE_LABEL == ":docker: Deploy Manifests" ]]; then
+    echo "--- :docker: Removing tags for deleted branches"
+    for BRANCH_TAG in $(dockerbranchtags=$(token=$(curl -fs --retry 3 'https://auth.docker.io/token?service=registry.docker.io&scope=repository:authelia/authelia:pull' | jq -r .token) && \
```

Comment:
what is this kind of token that does not require credentials?

Reply:
Similar to visiting DockerHub, the v2 API requires an auth workflow, so it's like an "unauthenticated token" which allows us to list all the tags.